src/modules/bible-reading/ui/views/reading-details.tsx

```
@@ -0,0 +1,141 @@
+        {/* Bài đọc khác */}
+        <div className="mt-8">
+          <h2 className="text-xl font-bold mb-4">Bài đọc khác</h2>
+          <div className="grid grid-cols-2 sm:grid-cols-3 gap-4">
+            {relatedReadings.map((reading) => (
+              <Link
+                href={`/bible-readings/${reading.id}`}
+                className="flex flex-col gap-2 overflow-hidden"
+              >
+                <ReadingCard
+                  key={reading.id}
+                  id={reading.id}
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)